clients/src/main/java/org/apache/kafka/clients/producer/internals/DefaultPartitioner.java

```diff
@@ -68,6 +70,18 @@ public int partition(String topic, Object key, byte[] keyBytes, Object value, by
         }
     }
 
+    private int getNextValue(String topic) {
```

Comment:
In Kafka, we generally avoid the `get` prefix. So `nextValue` would probably be better.